librz/util/calc.c

```diff
@@ -133,6 +133,25 @@ static inline RzNumCalcValue Ndiv(RzNumCalcValue n, RzNumCalcValue v) {
 	return n;
 }
 
+static inline RzNumCalcValue Nexp(RzNumCalcValue n, RzNumCalcValue v) {
+	RzNumCalcValue exp_n = n;
+	if (v.d - (int)v.d) {
+		RZ_LOG_WARN("floating point powers not yet supported")
+	}
+	if ((int)v.d > 0) {
+		for (int i = 1; i < (int)v.d; i++) {
+			n = Nmul(exp_n, n);
+		}
+	} else if ((int)v.d < 0) {
+		for (int i = -1; i >= (int)v.d; i--) {
+			n = Ndiv(n, exp_n);
+		}
```

Comment:
this seems a bit wrong. Try to play a bit with negative exponents, I see wrong results:
```
? 10**-5
```

To compute the negative I would do `1 / Nexp(n, -exp)` or something equivalent.